basic-bootc.ks.in

```diff
@@ -0,0 +1,20 @@
+# Substitute something in for REPO or this will all come crashing down.
+bootc --source-imgref=registry:quay.io/fedora-testing/fedora-bootc:rawhide-standard --stateroot=default
```

Comment:
Could you use the `KSTEST_OSTREECONTAINER_URL` variable here? It would allow to run the test on other variants (RHEL/CentOS Stream) in the future.